src/jit/lsra.cpp

```diff
@@ -3358,8 +3358,9 @@ bool LinearScan::isRefPositionActive(RefPosition* refPosition, LsraLocation refL
 //    False - otherwise
 //
 // Notes:
-//    This helper is designed to be used only from allocateBusyReg().
-//    The caller must have already checked for the case where 'refPosition' is a fixed ref.
+//    This helper is designed to be used only from allocateBusyReg(), where:
+//    - This register was *not* found when looking for a free register, and
+//    - The caller must have already checked for the case where 'refPosition' is a fixed ref.
```

Comment:
"the caller must have already checked "

Can you assert this?

Reply:
That's the first assert in the method. I've added that note to the comment.